style.css

```diff
@@ -0,0 +1,231 @@
+  background-size: cover;
+}
+
+.main-bg h1 {
+  color: var(--hoverColor);
+}
+
+.main-heading {
+  padding-top: 4em;
+}
+
+#nav-toggle {
+  color: var(--textColor);
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.social-icons {
+  color: var(--secondaryColor);
+  font-size: 1em;
+  font-weight: bold;
+  padding-right: 5px;
+}
+
+.lang {
+  padding-right: 5px;
+  list-style: none;
+}
+
+.nav-item {
+  place-self: center;
+  padding-right: 2em;
+}
+
+.product-heading {
+  color: var(--secondaryColor);
+  place-self: center;
+  border-bottom: 2px solid var(--hoverColor);
+  line-height: 50px;
+}
+
+.about-heading {
+  place-self: center;
+  border-bottom: 2px solid var(--hoverColor);
+  line-height: 50px;
+}
+
+.feature-post {
+  border-bottom: 2px solid var(--hoverColor);
+  line-height: 50px;
+}
+
+.past-event-card {
+  max-width: 25rem;
+  border: none;
+}
+
+.about-logo {
+  max-width: 400px;
+  border: 1px solid rgb(226, 225, 225);
+  box-shadow: 1px 1px gray;
+}
+
+.social-item {
+  padding-right: 1.5rem;
+  list-style: none;
+}
+
+.nav-link {
+  color: var(--secondaryTextColor);
+  text-decoration: none;
+  font-weight: bold;
+  font-size: medium;
+}
+
+.active {
+  color: var(--hoverColor);
+}
+
+.nav-link:hover {
+  color: var(--hoverColor);
+}
+
+.nav-btn {
+  color: var(--hoverColor);
+  border: 3px solid var(--hoverColor);
+  padding: 8px;
+  margin-top: 5px;
+}
+
+.nav-btn1 {
+  color: var(--hoverColor);
+  border: 3px solid var(--hoverColor);
+  padding: 8px;
+  margin-top: 10px;
+  margin-right: 1.5rem;
+}
+
+.footer-row {
+  display: flex;
+  justify-content: center;
+}
+
+.partner {
+  display: flex;
+  justify-content: center;
+}
+
+#nav-btn:hover {
+  color: var(--textColor);
+  text-decoration: none;
+  transition: 0.5s;
+  background-color: var(--hoverColor);
+  padding: 8px;
+}
+
+#nav-cc-btn {
+  margin: auto;
+}
+
+@media (min-width: 768px) {
```

Comment:
- [ ] You are encouraged to use mobile-first approach for your media queries i.e 
media queries are used to change the design from mobile to desktop, not the other way around.